plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml

```diff
@@ -44,6 +46,20 @@
                  style="push">
               <parameter
                     name="newWizardId"
+                    value="com.google.cloud.tools.eclipse.appengine.newproject.AppEngineFlex">
```

Comment:
Would it make sense to move the _New Flex_ as a menuContribution within the`.flex` fragment?  It would keep it all self-contained.
```
<menuContribution locationURI="menu:com.google.cloud.tools.eclipse.appengine.actions.new">
   <command commandId="…" etc />
</menuContribution>
```